### PR TITLE
Add: EPSS scoring info in CVEs

### PR DESCRIPTION
**Pull request by timopollmeier**
## What
Exploit Prediction Scoring System (EPSS) info is added to CVE info if it is available.

## Why
This provides information on the probabily of exploitation activity for vulnerabilities.

## References
GEA-558

### CODE REVIEW

**Comment by cfi-gb** [comment]
Cross-referencing the GSA part:
- greenbone/gsa#4033

File: src/manage_sql_secinfo.c [hunk 1]
```diff
@@ -2762,6 +2802,174 @@ update_scap_cves ()
...
+
+  quoted_cve = sql_quote (cve);
+  g_string_append_printf (inserts->statement,
+                          "%s ('%s', %lf, %lf)",
+                          first ? "" : ",",
+                          quoted_cve,
+                          epss,
+                          percentile);
+
+  inserts->current_chunk_size++;
```
Comment (a-h-abdelsalam) [inline src/manage_sql_secinfo.c:2828]:
```suggestion
  inserts->current_chunk_size++;
  g_free (quoted_cve);
```

File: src/manage_sql_secinfo.c [hunk 2]
```diff
@@ -2762,6 +2802,174 @@ update_scap_cves ()
...
+
+      insert_epss_score_entry (&inserts,
+                               cve_json->valuestring,
+                               epss_json->valuedouble,
+                               percentile_json->valuedouble);
+    }
+
+  inserts_run (&inserts, TRUE);
+  sql_commit ();
+  cJSON_Delete (parsed);
```
Comment (a-h-abdelsalam) [inline src/manage_sql_secinfo.c:2942]: You may want to free `list_item`
```suggestion
  cJSON_Delete (parsed);
  cJSON_Delete (list_item);
```

File: src/manage_sql_secinfo.c [hunk 3]
```diff
@@ -2762,6 +2802,174 @@ update_scap_cves ()
...
+
+  return 0;
+
+fail_insert:
+  inserts_free (&inserts);
+  sql_rollback ();
+  char *printed_item = cJSON_Print (list_item);
+  g_message ("%s: invalid item: %s", __func__, printed_item);
+  free (printed_item);
+  cJSON_Delete (parsed);
```
Comment (a-h-abdelsalam) [inline src/manage_sql_secinfo.c:2952]:
```suggestion
  cJSON_Delete (parsed);
  cJSON_Delete (list_item);
```

Comment (timopollmeier) [reply src/manage_sql_secinfo.c:2953]: Should be freed with `parsed` as above.

**Review by a-h-abdelsalam — APPROVED** [review]

**Pull request merged**